uqcsbot/scripts/repo.py

```diff
@@ -1,10 +1,58 @@
 from uqcsbot import bot, Command
 
+repos = {
+    "coc": "https://github.com/UQComputingSociety/code-of-conduct",
```

Comment:
Can this be explicitly "Code of Conduct"

Reply:
As "code-of-conduct" or "code_of_conduct"?